kedro/runner/runner.py

```diff
@@ -83,7 +83,9 @@ def run(
         """
 
         hook_or_null_manager = hook_manager or _NullPluginManager()
-        catalog = catalog.shallow_copy()
+
+        # Seems like it's not needed even for DataCatalog
+        # catalog = catalog.shallow_copy()
```

Comment:
Will that stay in? Or just for the PoC?

Reply:
I should go IMO as it doesn't seem useful, but I was going to double-check why it was there